packages/perseus/src/widgets/label-image.tsx

```diff
@@ -838,6 +876,23 @@ const styles = StyleSheet.create({
     choicesPopover: {
         padding: 0,
     },
+
+    pill: {
+        padding: "0 1em",
```

Comment:
If we want the padding around the text to have a consistent look when the font size changes (in the case of some accessibility tool), we'd want to use ems for a consistent ratio. Using rems or pxs would keep them static and would make it weird.

In most cases, this will never change. The padding will only ever change relative to the text it pads, which will never change. It's just a best practice I like to do.